src/components/look-controls.js

```diff
@@ -187,13 +187,27 @@ module.exports.Component = registerComponent('look-controls', {
     var pitchObject = this.pitchObject;
     var yawObject = this.yawObject;
     var sceneEl = this.el.sceneEl;
+    var q = this.hmdQuaternion;
 
     // In VR mode, THREE is in charge of updating the camera rotation.
     if (sceneEl.is('vr-mode') && sceneEl.checkHeadsetConnected()) { return; }
 
     // Calculate polyfilled HMD quaternion.
     this.polyfillControls.update();
-    hmdEuler.setFromQuaternion(this.polyfillObject.quaternion, 'YXZ');
+
+    // If rotation component exists, get its values and apply them.
+    var rotationComponent = el.components.rotation;
```

Comment:
There's always a rotation component

Reply:
I thought that thanks to @ngokevin, that is no longer true?

Reply:
Yup, this check is necessary now. Really don't need to go thru rotation component at all, just modify object3D directly.